tests/instances.spec.ts

```diff
@@ -228,3 +230,21 @@ test("instance yaml edit", async ({ page }) => {
 
   await deleteInstance(page, instance);
 });
+
+test("instance terminal operations", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  await visitAndStartInstance(page, instance);
+  await page.getByTestId("tab-link-Terminal").click();
+  await expect(page.locator(".xterm-screen")).toBeVisible(TIMEOUT);
+  const terminalInput = page.getByRole("textbox", { name: "Terminal input" });
```

Comment:
I think we can rely on autofocus here and directly use `page.keyboard.type`. Wdyt?

Reply:
that's a good point, we would test the focus logic as well then :+1: